package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
-    "build": "docusaurus build",
+    "build": "docusaurus build --no-minify",
```

Comment:
is this fr development? 

maybe we can add a new script : `"build-dev": "docusaurus build --no-minify",`

Reply:
this is because when I use some elements the build is fail

Reply:
This form is equal what like
we do on eos cr website